src/main/java/com/iota/iri/controllers/TransactionViewModel.java

```diff
@@ -712,11 +778,18 @@ public void setMetadata() {
                 : TransactionViewModel.FILLED_SLOT;
     }
 
-    public static void updateSolidTransactions(Tangle tangle, Snapshot initialSnapshot, final LinkedHashSet<Hash> analyzedHashes)
-            throws Exception {
+    /**
+     * Update solid transactions
+     * @param tangle Tangle
+     * @param initialSnapshot Initial snapshot
+     * @param analyzedHashes analyzed hashes
+     * @throws Exception Exception
+     */
+    public static void updateSolidTransactions(Tangle tangle, Snapshot initialSnapshot,
+            final LinkedHashSet<Hash> analyzedHashes) throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using implementation types like 'LinkedHashSet'; use the interface instead](https://app.codacy.com/app/GalRogozinski/iri/pullRequest?prid=4567264)